infinispan-remote/src/test/java/org/hibernate/ogm/datastore/infinispanremote/test/query/InfinispanRemoteJPQLBookTest.java

```diff
@@ -0,0 +1,103 @@
+package org.hibernate.ogm.datastore.infinispanremote.test.query;
+
+import static org.fest.assertions.Assertions.assertThat;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.hibernate.ogm.datastore.infinispanremote.utils.InfinispanRemoteServerRunner;
+import org.hibernate.ogm.utils.OgmTestCase;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.junit.runner.RunWith;
+
+@RunWith(InfinispanRemoteServerRunner.class)
+public class InfinispanRemoteJPQLBookTest extends OgmTestCase {
```

Comment:
Do you need all these tests? We already have the one in SimpleQueriesTest and they don't seem to add anything new

Reply:
This a temporary test... on to enable the development. In the final PR, will not be there.

Reply:
If you need them while developing you can extends SimpleQueriesTest instead of OgmTestCase or you can  update InfinispanRemoteBackendTckHelper and run the tests from there

Reply:
> This a temporary test... on to enable the development. In the final PR, will not be there.

I think it's better if you work using the existing tests and not spend time re-inventing the wheel, But feel free to work as you prefer.